### ISSUE TITLE
Link to definition of a priori insecure in MIX is no longer correct

### ISSUE THREAD

**Issue by martijnc** [w3c/webappsec-epr#1]
Section 4.3 links to a _a priori insecure_ definition in the mixed content specification but the term is no longer defined there.

I think this should be (not) a _a priori authenticated URL_ now?

**Comment by mikewest** [comment]
Yeah, I updated MIX in response to w3c/webappsec-mixed-content#1. This spec, and probably a few others, are still using the old language. I need to fix them.
